.gitlab/.pre/gitlab_configuration.yml

```diff
@@ -23,16 +23,19 @@ test_gitlab_compare_to:
     - pip install -r tasks/requirements.txt
     - inv pipeline.compare-to-itself
 
+.compute_gitlab_ci_config_rules:
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
I feel like this is more readable when the name of the rule describe the situation that should trigger the rule. What do you think about naming this: `except_not_push` or something like that?

Reply:
I wanted to include `compute_gitlab_ci_config` within the name of the rule because the goal is to use it when we depend on this job such that none of the jobs depending on it can run without it.
If at some point we add a job that can be run with a schedule, the rules can be changed without changing the name (thus the `!reference` usage).

Do you think it is better to remove it or rename it ?

Reply:
I see. What I find a bit misleading is that this rule is not self-sufficient. If you put only that rule your job will never be created since the rule specify when the job should never be created but not when the job should be created.
With that name I feel like we could think that you can just put that rule on the job if you want the job to be executed when the `compute_gitlab_ci_config` job is executed. Which is not the case.
But for sure having a common rule shared between all the jobs is a good idea. I do not really have a better suggestion for the naming so you can keep it as is ^^